src/typing/typeloadFields.ml

```diff
@@ -576,7 +591,13 @@ let build_fields (ctx,cctx) c fields =
 	let get_fields() = !fields in
 	let pending = ref [] in
 	c.cl_build <- (fun() -> BuildMacro pending);
-	build_module_def ctx (TClassDecl c) c.cl_meta get_fields cctx.context_init (fun (e,p) ->
+	let meta =
```

Comment:
Can you try if instead of doing this we can add `Meta.Using` to the list of metas at typeloadModule.ml line 325?